machines/netcup-x86-runner-1/configuration.nix

```diff
@@ -2,7 +2,10 @@
 { pkgs, lib, ... }: {
 
   lounge-rocks = {
-    netcup-x86.enable = true;
+    cloud-provider = {
+      enable = true;
+      netcup.enable = true;
+    };
```

Comment:
Same as above. Consider dropping  `cloud-provider.enable` in favor of just having `cloud-provider.netcup.enable`